### ISSUE TITLE
Floating "dlclose"

### ISSUE THREAD

**Issue by SoniEx2** [bearcove/feedback#55]
**Where's the problem at?**
Please link to the article
https://fasterthanli.me/articles/so-you-want-to-live-reload-rust#what-can-prevent-from-unloading-a-library

**No, seriously, where is it?**
Copy/paste the paragraph the problem is in, maybe highlight what's wrong?
No really, we linked the relevant anchor. Tho it's actually right *above* it.

**Describe the issue and a fix if you know it**
In plain English, explain why this is wrong, and how to fix it (if you know how).
There's a floating box that says "dlclose" on the left on a 1080p screen with firefox desktop on x86_64 Linux. (We don't know if we need to be that specific, but hey.)

**Comment by Janrupf** [comment]
Also happens on Chromium x86_64 Linux:
![Floating box](https://img.janrupf.net/i/5zblw.png)

**Comment by Janrupf** [comment]
(On further inspection, this seems to be a duplicate of https://github.com/bearcove/feedback/issues/54)

**Comment by fasterthanlime** [comment]
Closing as duplicate of #54

**Comment by fasterthanlime** [comment]
Fixed it:

<img width="539" alt="image" src="https://user-images.githubusercontent.com/7998310/167495489-7f47f15c-86cb-42de-a77a-1d7f79461839.png">

<img width="741" alt="image" src="https://user-images.githubusercontent.com/7998310/167495509-f7c93aed-e091-4208-9140-2779fb1f2e21.png">

https://fasterthanli.me/articles/so-you-want-to-live-reload-rust#what-can-prevent-dlclose-from-unloading-a-library
